### PR TITLE
Fixing ops health check skip and checking of ES version for not ready pods

### PR DESCRIPTION
**Pull request by ewolinetz**
This PR contains the following changes:

* Fixing evaluating if ops deployment needs to skip health check -- We were incorrectly not evaluating that we had created new ES ops nodes and would still wait for the pod to be ready before continuing to rolling out the next one even when we didn't want to (causes issues in logging clusters > 3 nodes)

* Removing logic for determining version -- the intention for this logic was never actually used and instead makes it difficult to use custom built images locally and requires constant updating.

* Fixing pod check for elasticsearch to get running version -- we currently would only check if the pod is in a "Running" state, which was incorrect since it could be running and the ES container was not yet ready and would prevent us from getting the current version via a curl. Addresses https://bugzilla.redhat.com/show_bug.cgi?id=1550140

### CODE REVIEW

File: roles/openshift_logging_elasticsearch/tasks/get_es_version.yml [hunk 1]
```diff
@@ -1,23 +1,47 @@
...
+  when:
+  - ops_pod_list.results.results is defined
+  - item.status.phase == "Running"
+  - item.status.containerStatuses[0].ready == true
+  - item.status.containerStatuses[1].ready == true
 
 - name: "Getting ES version for logging-es-ops cluster"
   command: >
-    {{ openshift_client_binary }} exec {{ _ops_cluster_pods.stdout.split(' ')[0] }} -c elasticsearch -n {{ openshift_logging_elasticsearch_namespace }} -- {{ __es_local_curl }} -XGET 'https://localhost:9200/'
+    {{ openshift_client_binary }} exec {{ available_ops_pod }} -c elasticsearch -n {{ openshift_logging_elasticsearch_namespace }} -- {{ __es_local_curl }} -XGET 'https://localhost:9200/'
```
Comment (jcantrill) [inline roles/openshift_logging_elasticsearch/tasks/get_es_version.yml]: If you don't mind relying on an image util
```
oc exec -c elasticsearch -n $NAMESPACE -- es_util --query=/
```
this give you the admin certs.  you can addtionally pass all curl args after 'query'

Comment (ewolinetz) [reply roles/openshift_logging_elasticsearch/tasks/get_es_version.yml]: lets take that up in another pr... i think there's more to clean up if we do that

**Comment by sdodson** [comment]
That's a CI flake.

**Comment by ewolinetz** [comment]
of course this wasn't a clean cherry-pick...

**Pull request merged**